frontend/benefit/handler/src/components/header/useHeader.ts

```diff
@@ -0,0 +1,56 @@
+import useLocale from 'benefit/handler/hooks/useLocale';
```

Comment:
Even this could be in shared folder if useTranslation comes from `next-i18next` and `getLanguageOptions` is also moved to shared? I think we have this same header login already copy-pasted three times per each service but the code is actually 99% same.

Reply:
fixed